_examples/ThisExtensionFunc.md

```diff
@@ -0,0 +1,62 @@
+---
+title: this in extension functions
+---
+    
+<div class="sample" markdown="1">
+
+Normally, when you create a class, by default it inherits from *Any*, the Kotlin default supertype which declares toString(), hashCode() and equals().
```

Comment:
looks good, a tiny suggestion:
Normally, when you create a class, it inherits from **Any** (the Kotlin default supertype). **Any** declares toString(), hashCode(), and equals().

Reply:
Good point.